### DISCUSSION TITLE
Query sum of a connected record in includes object

### DISCUSSION THREAD

**Discussion by n-glaz** [prisma/prisma#15580] [Q&A] [answered]
Hi, I have a Receipt record that has many Items, and I want to return a `_sum` of the cost of every Item in a single query of the Receipt, something like:

```typescript
const receipt = await prisma.receipt.findUnique({
  where: {
       id: receiptId
    }
  },
  include: {
      _sum: {
           items: {  cost: true }
      }
   }
```

Is this possible in the current version of Prisma?

**Comment by austincrim** [comment, marked as answer]
Hey @n-glaz 👋,

I don't think it will work querying through the `receipt` model, but it may work going the other direction.

```ts
const result = await prisma.item.aggregate({
  where: { receiptId: 'someId' },
  _sum: { cost: true }
})
```

If that's not the result you want, I would encourage you to explain your use case in a new [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
